lib/UtapiClient.js

```diff
@@ -7,9 +7,25 @@ const werelogs = require('werelogs');
 const { errors } = require('arsenal');
 const Datastore = require('./Datastore');
 const { generateKey, generateCounter, generateStateKey } = require('./schema');
-const redisClient = require('../utils/redisClient');
+const RedisClientv2 = require('../libV2/redis');
 const member = require('../utils/member');
 
+function redisClientv2(conf, log) {
```

Comment:
May be add jsdoc for `redisClientv2`, it will be easier to check what's required in `conf`!